hazelcast/src/test/java/com/hazelcast/map/MapLockTest.java

```diff
@@ -386,6 +390,28 @@ public void run() throws Exception {
         }, 30);
     }
 
+    /**
+     * See issue #4888
+     */
+    @Test
+    public void lockStoreShouldBeRemoved_whenMapIsDestroyed() {
+        HazelcastInstance hz = createHazelcastInstance();
+        IMap map = hz.getMap(randomName());
+        for (int i = 0; i < 1000; i++) {
+            map.lock(i);
```

Comment:
Original issue is, there are leaking empty `LockStore`s after map is destroyed. During destroy, `LockStore`s are already purged, all entries are removed. But `LockStore` itself remains in the lock registry. 

So, it doesn't matter, unlock is called or not. If it makes the test easier to understand, I can remove the unlocks.